### ISSUE TITLE
NTR: Cortical tubers

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4058]
Synonyms: HP:0009717 
; HPO ID: NA 
; Alternative ID: Abnormality of the cerebral cortex  
; Son of_1: Cerebral hamartomata 
; Textual definition:  Cortical tubers in the brain are hamartomatous lesions typically located at the gray-white matter interface, commonly in the frontal and parietal lobes. Cortical tubers are composed of abnormal glial and neural cells, and the size, number, and location vary among patients. 
; PMID to support changes:  
; label: NA 
; Curator: 191100 613254 
; OMIM -disease: 605284 191092 
; OMIM -gene

**Comment by pnrobinson** [comment]
I will transfer the disease annotations to the hpo annot repo. The term already exists

https://hpo.jax.org/app/browse/term/HP:0009717
